definition/alertmanager_validation.go

```diff
@@ -82,14 +85,29 @@ func (r *Route) ValidateReceivers(receivers map[string]struct{}) error {
 	return nil
 }
 
-func (r *Route) ValidateMuteTimes(muteTimes map[string]struct{}) error {
+func (r *Route) ValidateMuteTimes(timeIntervals map[string]struct{}) error {
 	for _, name := range r.MuteTimeIntervals {
-		if _, exists := muteTimes[name]; !exists {
+		if _, exists := timeIntervals[name]; !exists {
 			return fmt.Errorf("mute time interval '%s' does not exist", name)
 		}
 	}
 	for _, child := range r.Routes {
-		err := child.ValidateMuteTimes(muteTimes)
+		err := child.ValidateMuteTimes(timeIntervals)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (r *Route) ValidateActiveTimes(timeIntervals map[string]struct{}) error {
```

Comment:
We're not using this method anywhere.
We could change `ValidateMuteTimes -> ValidateTimeIntervals` and check both `MuteTimeIntervals` and `ActiveTimeIntervals` in the same place.

Reply:
We use it in [grafana/grafana](https://github.com/grafana/grafana/blob/7b919e32771b8228684234d5cc7262d915a96534/pkg/services/ngalert/provisioning/notification_policies.go#L83), but I agree one method would be better. I'll add it as a 2nd method for now so we don't needlessly break grafana/grafana compilation on dependency update, then we can remove at will.

Reply:
Ah ok! The plan is to use `ValidateActiveTimes` in grafana/grafana right next to the call to `ValidateMuteTimes` then. Sounds good 🙂